src/main/java/data/ScriptingProject.java

```diff
@@ -0,0 +1,84 @@
+package data;
+
+import lombok.Getter;
+import lombok.Setter;
+import xml.XmlReader;
+import xml.XmlWriter;
+
+import java.util.ArrayList;
+
+
+
+
+/**
+ * Created by Bart.
+ * Class to store top-level properties of a scripting project.
+ */
+public class ScriptingProject {
+
+    // Description of this project
+    @Getter
+    private String description;
+
+    // List of cameras that are available in this project
+    @Getter @Setter
+    private ArrayList<Camera> cameras;
```

Comment:
A more abstract type would give more freedom on choosing a final datastructure to store the data in, would it not? Maybe a collection? The collection is implemented by a lot of datastructures, including the `ArrayList`.